src/Persistence/configuration.py

```diff
@@ -0,0 +1,31 @@
+from Persistence.abstractconfigstore import ConfigStore
+
+
+class Configuration:
+
+    config_store = None
+
+    def __init__(self, config_store: ConfigStore):
```

Comment:
Does the __ suffix actually changes this to a private method.
Or is it just an indication?

Reply:
Python doesn't have private methods. The __ adds name mangling to avoid clashes with subclasses; IDEs can also pick it up as a hint.  For __init__ it simply denotes the default constructor (one of the "magic methods" you don't want to touch)